lib/dangermattic/plugins/view_code_pr_checker.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Danger
+  # Plugin to detect View files in a PR but without having accompanying screenshots.
+  class ViewCodePRChecker < Plugin
```

Comment:
I wonder if we couldn't pick a more explicit name for that plugin. Just reading `ViewCodePRChecker` feels like it could equally be a plugin that allows you to "View" information about your "Code" in your "PR" (like stats about code changes and stuff).

Maybe `ViewChangesNeedScreenshots`, or `ViewRelatedChangesHasScreenshots < Plugin`, or something like that?

Reply:
Great suggestion, yesterday when reviewing the code I also thought the same. `ViewChangesNeedScreenshots` sounds good to me!